src/main/java/org/sonatype/repository/chef/internal/AssetKind.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.chef.internal;
+
+import javax.annotation.Nonnull;
+
+import org.sonatype.nexus.repository.cache.CacheControllerHolder;
+import org.sonatype.nexus.repository.cache.CacheControllerHolder.CacheType;
+
+/**
+ * Asset kinds for Chef
+ *
+ * Examples of all can be found at: https://github.com/kubernetes/helm/blob/master/docs/chart_repository.md#the-chart-repository-structure
+ * @since 0.0.1
+ */
+public enum AssetKind
+{
+  COOKBOOK_DETAIL_VERSION(CacheControllerHolder.METADATA),
+  COOKBOOK_DETAILS(CacheControllerHolder.METADATA),
+  COOKBOOKS_LIST(CacheControllerHolder.METADATA),
+  COOKBOOKS_SEARCH(CacheControllerHolder.METADATA),
+  COOKBOOK(CacheControllerHolder.CONTENT);
```

Comment:
Trivial: Not sure CacheControllerHolder provides any additional information here, statically import it?

Reply:
In 685f877